reinforcement_learning/trainer/agent.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
is this file used?

Reply:
This defines an interface, in a later PR DDPG, TD3 and C2A2 all implements this interface.

Reply:
`# Copyright 2018 Google LLC`